<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>65</y>
     <width>430</width>
     <height>570</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>22</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;Trebuchet MS&quot;;
background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border: 6px solid lightgreen;</string>
   </property>
   <property name="title">
    <string>Tasks</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="gr_to_aut">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>250</width>
      <height>70</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover {
    background-color:lightgreen;
	color: black;
}

QPushButton{
font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border:3px;
border-radius:10px;
}
</string>
    </property>
    <property name="text">
     <string>Grammar -&gt; Automata</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="aut_to_gr">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>250</width>
      <height>70</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover {
    background-color:lightgreen;
	color: black;
}


QPushButton{
font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border:3px;
border-radius:10px;
}
</string>
    </property>
    <property name="text">
     <string>Automata -&gt; Grammar </string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="morsiphier_btn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>400</y>
      <width>250</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover {
    background-color:lightgreen;
	color: black;
}


QPushButton{
font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border:3px;
border-radius:10px;
}
</string>
    </property>
    <property name="text">
     <string>Morsiphier</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>660</y>
     <width>431</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel:hover{
background-color: black;
color:red;
}
QLabel{
}</string>
   </property>
   <property name="text">
    <string>Created by Samandar Yokubov</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
